Gemfile

```diff
@@ -246,7 +246,7 @@ unless ENV["APPLIANCE"]
     gem "factory_bot",      "~>4.11.1", :require => false
     gem "faker",            "~>1.8",    :require => false
     gem "timecop",          "~>0.7.3",  :require => false
-    gem "vcr",              "~>3.0.2",  :require => false
```

Comment:
I believe we need to update webmock 👇 to 3.6.0+ to fix the hash diff change

See: https://github.com/bblimke/webmock/blob/master/CHANGELOG.md#360

Reply:
Hm, VCR has a dependency on hashdiff as well: https://github.com/vcr/vcr/blob/master/vcr.gemspec#L34

Reply:
@jrafanie Ok, I've also added an update for webmock.